gui/middleware/zfs.py

```diff
@@ -291,18 +288,39 @@ class Vdev(Tnode):
     def __repr__(self):
         return "<Section: %s>" % self.name
 
+    def get_disks(self):
+        disks = []
+        for child in self:
+            if isinstance(child, Vdev):
+                disks.extend(child.get_disks())
+            else:
+                if child.disk:
```

Comment:
I am checking the disk here if it is None or not?

Reply:
Would this work?

Reply:
I deleted my comment a few minutes ago ; )

Reply:
Oh : )